### ISSUE TITLE
Multiple beneficaries

### ISSUE THREAD

**Issue by 3esmit** [status-im/commiteth#161]
### User Story
As a  user I want to be able to get partial bounties in case multiple users contributed in PR

**Comment by arash009** [comment]
Can we also extend this requirement to include getting partial bounties in case my PR doesnt fully solve the issue but has got it x% closer and I want to stop here.

**Comment by 3esmit** [comment]
Yes, we could add that, but I'm not sure how SOB would be able to check that.
